sdk/core/Azure.Core/src/Pipeline/RetryMode.cs

```diff
@@ -3,9 +3,21 @@
 
 namespace Azure.Core.Pipeline
 {
+    /// <summary>
```

Comment:
I added these when I was considering using the `Core` types directly and then forgot that I did so.   While I didn't intend for the doc comments to be included here, I'm inclined to consider them helpful and leave them hanging around.